wolfSSL/src/chapter02.md

```diff
@@ -484,6 +648,22 @@ Enables SHA-384 support.
 
 Enables SHA-512 support.
 
+#### WOLFSSL_AES_128
+
+Enables AES-128 support.
+
+#### WOLFSSL_AES_256
+
+Enables AES-256 support.
+
+#### WOLFSSL_AES_DIRECT
+
+The AES option is used when you want to implement AES counter mode and when you want to use pkcs7. It's primarily used to make a direct call to a function in user settings, providing a flexible and customizable AES implementation.
```

Comment:
Description is not correct. It it used when direct AES ECB mode API's should be enabled and exposed. It is used by PKCS7.